subsys/bluetooth/services/dis.c

```diff
@@ -179,7 +179,7 @@ static ssize_t read_system_id(struct bt_conn *conn, const struct bt_gatt_attr *a
 #endif
 
 #if CONFIG_BT_DIS_UDI
-#define DIS_STR_SIZE(x)           ((x[0]) != '\0' ? strlen(x) + sizeof('\0') : 0)
+#define DIS_STR_SIZE(x)           ((x[0]) != '\0' ? strlen(x) + 1 : 0)
```

Comment:
Out of curiosity, why did you opt for `sizeof("")` one place, and just `1` here, when they, as far as I can tell, refer to the same (i.e. size of the NULL terminator)?